### PR TITLE
Update build instructions to use localhost ip

### PR DESCRIPTION
**Pull request by LanderU**
Signed-off-by: LanderU <lander.usategui@gmail.com>

### CODE REVIEW

**Review by carlossvg — APPROVED** [review]

**Review by razr — COMMENTED** [review]
Hi, @LanderU I think both options are valid. One can use 127.0.0.1 to access an HTTP server *locally*. And to use ```<your machine IP>``` to access an HTTP server *remotely*.

File: README.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ To run the site locally, type the following commands at the top-level of the rep
 
 ```bash
 cd build
-python3 -m http.server --bind <your machine IP>
+python3 -m http.server --bind 127.0.0.1
 ```
 
-It will be accessible remotely at http://<your machine IP>:8000/index.html
+It will be accessible remotely at http://127.0.0.1:8000
```
Comment (LanderU) [inline README.md]:
```suggestion
It will be accessible remotely at http://<remote_ip_address>:8000
```

**Comment by LanderU** [comment]
@razr, you're right, I changed it to <remote_ip_address>

**Review by razr — APPROVED** [review]
lgtm

**Pull request merged**